redwood-layout-uiview/src/commonMain/kotlin/app/cash/redwood/layout/uiview/YogaUIView.kt

```diff
@@ -44,12 +44,11 @@ internal class YogaUIView(
   }
 
   override fun intrinsicContentSize(): CValue<CGSize> {
-    return calculateLayoutWithSize(CGSizeMake(Size.UNDEFINED.toDouble(), Size.UNDEFINED.toDouble()))
+    return calculateLayoutWithSize(Size.UNDEFINED, Size.UNDEFINED)
   }
 
   override fun sizeThatFits(size: CValue<CGSize>): CValue<CGSize> {
-    val constrainedSize = size.useContents { sizeForConstraints(this) }
```

Comment:
Rewriting the caller’s requested size was our previous mechanism to implement Constraint.Wrap. It didn’t work in special cases.